### PR TITLE
ACS-6826 test to apply prediction to updated node

### PR DESCRIPTION
**Pull request by azakrzewski-hy**
[ACS-6826](https://hyland.atlassian.net/browse/ACS-6826)
Upload a new version of a document and check that HxI receives the metadata update then ACS receives prediction for the node.
Added test covers scenario:
1. node is created
2. prediction is generated for the node from step 1.
3. node from step 1. is updated together with field that has prediction from step 2.
4. prediction for updated node (in step 3.) is generated but updated node should not apply this prediction and keep updated user value

[ACS-6826]: https://hyland.atlassian.net/browse/ACS-6826?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java [hunk 1]
```diff
@@ -89,6 +89,14 @@ public class UpdateNodeE2eTest
               }
             ]
             """;
+    private static final String UPDATE_NODE_PROPERTIES = """
+            {
+                "properties": {
+                    "cm:versionLabel": "1.1",
+                    "cm:title": "User title"
+                    }
```
Comment (tpage-alfresco) [inline e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java:97]:
```suggestion
                }
```

**Comment by azakrzewski-hy** [comment]
> The test looks fine. It looks like a very similar scenario to the existing test because the ACS update doesn't impact the prediction.

It is very similar to the existing test. Idea was to cover scenario for:
"Upload a new version of a document and check that HxI receives the metadata update"
If you say that there is no much value in adding this test we can drop it

**Comment by krdabrowski** [comment]
> "Upload a new version of a document and check that HxI receives the metadata update"
Adam if you would like to verify only this part, then maybe it's worth considering removing the redundant part - prediction mock publication and test only what you meant?

File: e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java [hunk 2]
```diff
@@ -25,17 +25,7 @@
...
-import static com.github.tomakehurst.wiremock.client.WireMock.containing;
-import static com.github.tomakehurst.wiremock.client.WireMock.exactly;
-import static com.github.tomakehurst.wiremock.client.WireMock.get;
-import static com.github.tomakehurst.wiremock.client.WireMock.givenThat;
-import static com.github.tomakehurst.wiremock.client.WireMock.moreThanOrExactly;
-import static com.github.tomakehurst.wiremock.client.WireMock.postRequestedFor;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlEqualTo;
-import static com.github.tomakehurst.wiremock.client.WireMock.urlPathTemplate;
-import static com.github.tomakehurst.wiremock.client.WireMock.verify;
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
```
Comment (krdabrowski) [inline e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java:28]: Avoid wildcard imports

**Review by krdabrowski — APPROVED** [review]
LGTM

File: e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java (hunk 2 above)
Comment (azakrzewski-hy) [reply e2e-test/src/test/java/org/alfresco/hxi_connector/e2e_test/UpdateNodeE2eTest.java]: Changed wildcard import for separate method imports

**Pull request merged**